packages/block-library/src/missing/edit.native.js

```diff
@@ -18,26 +18,86 @@ import { __ } from '@wordpress/i18n';
  */
 import styles from './style.scss';
 
+const isAndroid = Platform.OS === 'android';
+const platformText = isAndroid ? 'Android' : 'iOS';
 export class UnsupportedBlockEdit extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { showHelp: false };
+	}
+
+	toggleSheet() {
+		this.setState( {
+			showHelp: ! this.state.showHelp,
+		} );
+	}
+
+	renderHelpIcon() {
+		return <TouchableWithoutFeedback
+			accessibilityLabel={ __( 'Help icon' ) }
+			accessibilityRole={ 'button' }
+			accessibilityHint={ __( 'Tap here to show help' ) }
+			onPress={ this.toggleSheet.bind( this ) }
+		>
+			<View style={ styles.helpIconContainer } >
+				<Icon
+					className="unsupported-icon-help"
+					label={ __( 'Help icon' ) }
+					icon="editor-help"
+				/>
+			</View>
+		</TouchableWithoutFeedback>;
+	}
+
+	renderSheetButton( text, action ) {
+		return <TouchableOpacity
+			onPress={ action() }
+			style={ styles.sheetButton }
+		>
+			<Text style={ styles.sheetButtonText } >{ text }</Text>
+		</TouchableOpacity>;
+	}
+
+	renderSheet( title ) {
+		return <BottomSheet
```

Comment:
I'm surprised that it's not caught by the lint job, but I noticed a different JSX style in this file than the rest of the code: we normally wrap the returned JSX in parentheses and jump to a new line:

```jsx
return (
    <BottomSheet
        isVisible={ this.state.showHelp }
 	hideHeader
        onClose={ this.toggleSheet.bind( this ) }
    >
        { /* ... */ }
    </BottomSheet>
);
```

I can't find it mentioned in any coding style guide, but that's the style I've seen and written everywhere else.

Reply:
Addressed in f7c8582.